src/utils/modules.js

```diff
@@ -303,8 +303,7 @@ export class ModuleTracer extends Tracer {
         for (let i = 0; i < lengthValue.value; i++) {
           const elementValue = logger.tryQuery(
             () => Get(realm, ((value: any): ArrayValue), "" + i),
-            realm.intrinsics.undefined,
-            false
```

Comment:
This was causing a Flow error for me locally saying that too many arguments were passed. I guess this was triggered because I modified `Realm.js`.

Reply:
The `false` parameter is no longer needed. I removed it in an earlier pull request.